plugins/catalog-backend-module-azure/src/providers/AzureDevOpsEntityProvider.ts

```diff
@@ -166,17 +166,18 @@ export class AzureDevOpsEntityProvider implements EntityProvider {
   }
 
   private createLocationSpec(file: CodeSearchResultItem): LocationSpec {
+    const target = this.createObjectUrl(file);
+
     return {
       type: 'url',
-      target: this.createObjectUrl(file),
+      target: target,
       presence: 'required',
     };
   }
 
   private createObjectUrl(file: CodeSearchResultItem): string {
-    const baseUrl = `https://${this.config.host}/${this.config.organization}/${this.config.project}`;
-    return encodeURI(
-      `${baseUrl}/_git/${file.repository.name}?path=${file.path}`,
-    );
+    const baseUrl = `https://${this.config.host}/${this.config.organization}`;
```

Comment:
I'm unaware of which one is better in terms of code style. Is there any guidance on that? I'm happy to do ether :)

Reply:
Personally I like to change as little as possible. I know the existing code already works well so why change it outside of the one part you are actually changing?

Reply:
Yep, this ☝️, where did the `encodeURI` go? 😉